src/SSHDebugPS/PSOutputParser.cs

```diff
@@ -74,36 +56,36 @@ public string Extract(string line)
             }
         }
 
-        private const string UserNamePrefix = "CurrentUserName: ";
-        private const string PSCommandLine = "ps -axww -o pid=A,ruser=B,args=C";
-        public const string CommandText = "echo " + UserNamePrefix + "$USER; " + PSCommandLine;
+        // Use padding to expand column width. 10 for pid and 32 for userid
+        private const string PSCommandLineFormat = "ps{0}-o pid=ppppppppppp -o ruser=rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr -o args";
```

Comment:
You might want to add some notes here as to the problems you ran into with various "ps" implementations and the reason you settled on doing it this way so whoever touches this next doesn't have to rediscover it.